WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingsViewController.swift

```diff
@@ -342,8 +437,61 @@ open class NotificationSettingsViewController: UIViewController {
         }
     }
 
+    enum Section: Int {
+        case blog
+        case followedSites
+        case other
+        case wordPressCom
+
+        func headerText() -> String? {
+            switch self {
+            case .blog:
+                return NSLocalizedString("Your Sites", comment: "Displayed in the Notification Settings View")
+            case .followedSites:
+                return NSLocalizedString("Followed Sites", comment: "Displayed in the Notification Settings View")
+            case .other:
+                return NSLocalizedString("Other", comment: "Displayed in the Notification Settings View")
+            case .wordPressCom:
+                return nil
+            }
+        }
+
+        func footerText() -> String? {
+            switch self {
+            case .blog:
+                return NSLocalizedString("Customize your site settings for Likes, Comments, Follows, and more.",
+                                         comment: "Notification Settings for your own blogs")
+            case .followedSites:
+                return NSLocalizedString("Customize your followed site settings for New Posts and Comments",
+                                         comment: "Notification Settings for your followed sites")
+            case .other:
+                return nil
+            case .wordPressCom:
+                return NSLocalizedString("We’ll always send important emails regarding your account, " +
+                                         "but you can get some helpful extras, too.",
+                                         comment: "Title displayed in the Notification Settings for WordPress.com")
+            }
+        }
+
+        var showBadge: Bool {
+            switch self {
+            case .blog:
+                return true
```

Comment:
Is a check here needed for "feature flag on" && "app is WP"?

Reply:
Thanks for catching this @Guarani !! It's indeed missing, will fix it momentarily!